cmd/podman/images/prune.go

```diff
@@ -53,9 +53,7 @@ func init() {
 func prune(cmd *cobra.Command, args []string) error {
 	if !force {
 		reader := bufio.NewReader(os.Stdin)
-		fmt.Printf(`
-WARNING! This will remove all dangling images.
-Are you sure you want to continue? [y/N] `)
+		fmt.Printf("%s", createPruneWarningMessage(pruneOpts))
```

Comment:
```suggestion
		fmt.Print(createPruneWarningMessage(pruneOpts))
```
No reason to format string as a string.

Reply:
Lint is the reason. I believe the message was: 
```
printf-style function with dynamic format string and no further arguments should use print-style function instead (SA1006)go-staticcheck
```